go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.8
 require (
 	github.com/VictoriaMetrics/fastcache v1.12.1
 	github.com/antithesishq/antithesis-sdk-go v0.3.8
-	github.com/ava-labs/avalanchego v1.12.0-config-pebble-sync
+	github.com/ava-labs/avalanchego v1.12.1-0.20241217204724-0ae6f7cc90e4
```

Comment:
shouldn't we use v1.12.1 (probably a simple answer I'm not aware of! 😉)

Reply:
I think we had not released 1.12.1 at the time this PR was created, but seems we can use it now.